nflow-explorer/src/app/search/searchResult.html

```diff
@@ -3,23 +3,23 @@ <h2>Workflows</h2>
   <div class="alert alert-info" role="alert" ng-hide="ctrl.results.length">
     No workflow instances found
   </div>
-  <table ng-show="ctrl.results.length" id="search-result" class="table table-striped table-hover">
+  <table st-table="displayedResults" st-safe-src="ctrl.results" ng-show="ctrl.results.length" id="search-result" class="table table-striped table-hover">
     <thead>
-    <th>Id</th>
-    <th>Workflow type</th>
-    <th>State</th>
-    <th>State text</th>
-    <th>Status</th>
-    <th>Business key</th>
-    <th>External id</th>
-    <th>Retries</th>
-    <th>Created</th>
-    <th>Started</th>
-    <th>Modified</th>
-    <th>Next activation</th>
+    <th st-sort="id">Id</th>
```

Comment:
Would be nice if mouse cursor would indicate clickable header somehow?

Reply:
Table header looks clickable now. Also added styles for displaying sort order direction (didn't notice that before).